datafusion-cli/src/exec.rs

```diff
@@ -340,13 +340,10 @@ mod tests {
         let session_token = "fake_session_token";
         let location = "s3://bucket/path/file.parquet";
 
-        // Missing region
+        // Missing region, use object_store defaults
```

Comment:
object_store defaults now to us-east-1 https://github.com/apache/arrow-rs/pull/5244